### DISCUSSION TITLE
Profile

### DISCUSSION THREAD

**Discussion by Yorztib25** [community/community#42459] [Other Feature Feedback, Questions, & Ideas]
How create

**Comment by jfullstackdev** [comment]
create landing profile on GitHub ? the other one is landing page on GitHub Pages, which is a separate domain using `.github.io`

check this out 

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme

for GitHub Pages

https://github.com/jdevstatic/github-pages-tutorial

check out my landing profile README and the one on GitHub Pages, 

https://github.com/jdevstatic

https://jdevstatic.github.io/
